tests/jsx/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -228,6 +311,62 @@ exports[`hug.js 1`] = `
 
 `;
 
+exports[`logical-expression.js 1`] = `
+<div>
+  {a || "b"}
+</div>;
+
+<div>
+  {a && "b"}
+</div>;
+
+<div>
+  {a || <span></span>}
+</div>;
+
+<div>
+  {a && <span></span>}
+</div>;
+
+<div>
+  {a && <span>
```

Comment:
The snapshot contains both the unformatted and formatted code, with a line of `~` characters in-between them.

The formatted code is on [line 354](https://github.com/prettier/prettier/pull/2398/files/e5254e78f13518f488205789af560af12a7fcbb0#diff-4591a84e8b3362fa01bb47a987d3944cR354)